browser-extensions/common/js/lib/challenges.js

```diff
@@ -71,6 +71,20 @@ function generate_running_challenge_data(data) {
       "data": [
         {"month": 11, "day": 25}
       ],
+      "help": "Run a parkrun on the 25th of December."}))
+	 challenge_data.push(challenge_on_holidays(data, {
+      "shortname": "thanksgiving-turkey",
+      "name": "Thanksgiving Turkey",
+      "data": [
+	    // No USA parkruns before 2012
+		  {"fulldate": "22/11/2012"}, {"fulldate": "28/11/2013"}, {"fulldate": "27/11/2014"},
+		  {"fulldate": "26/11/2015"}, {"fulldate": "24/11/2016"}, {"fulldate": "23/11/2017"},
+		  {"fulldate": "22/11/2018"}, {"fulldate": "28/11/2019"}, {"fulldate": "26/11/2020"},
+		  {"fulldate": "25/11/2021"}, {"fulldate": "24/11/2022"}, {"fulldate": "23/11/2023"},
+		  {"fulldate": "28/11/2024"}, {"fulldate": "27/11/2025"}, {"fulldate": "26/11/2026"},
+		  {"fulldate": "25/11/2027"}, {"fulldate": "23/11/2028"}, {"fulldate": "22/11/2029"}
+		// More dates to be added in the future
+      ],
       "help": "Run a parkrun on the 25th of December."}))
```

Comment:
Update this when merging :-)
```suggestion
      "help": "Run a parkrun on Thanksgiving."}))
```